src/js/index.js

```diff
@@ -437,10 +441,18 @@ function handleTheme() {
   }
 }
 
+function toggleWorkspace(workspace) {
+  setLocalStorageItem("manifest_workspace", workspace);
+  location.reload();
+}
+
 function onKeydown(e) {
   if ((e.code === "KeyT" || e.keyCode === 84) && e.altKey) {
     toggleTheme();
   }
+  if (((e.keyCode >= 48 && e.keyCode <= 57) || (e.keyCode >= 96 && e.keyCode <= 105)) && e.altKey) {
+    toggleWorkspace(e.code.substr(-1));
+  }
```

Comment:
I'm getting all top numbers + numeric keypad and the `substr(-1)` is so `Digit3` becomes `3`